### ISSUE TITLE
New error entries with 1.14.0-rc.1

### ISSUE THREAD

**Issue by Dannhausen** [vmware-tanzu/velero#7892]
**What steps did you take and what happened:**
We did an update to version 1.14.0-rc.1 in our developer stage. After that update we got several log entries like this:
time="2024-06-14T00:44:51Z" level=error msg="error encountered while scanning stdout" backupLocation=velero/default cmd=/plugins/velero-plugin-for-aws controller=backup-sync error="read |0: file already closed" logSource="pkg/plugin/clientmgmt/process/logrus_adapter.go:90"

Our backup is running normal so far.

**What did you expect to happen:**
No new unknown error messages 

**Environment:**

- Velero version (use `velero version`): 
  Client:
        Version: v1.13.2
        Git commit: 4d961fb6fec384ed7f3c1b7c65c818106107f5a6
  Server:
        Version: v1.14.0-rc.1
- Velero features (use `velero client config get features`): not set 
- Kubernetes version (use `kubectl version`): Server Version: version.Info{Major:"1", Minor:"26", GitVersion:"v1.26.15+vmware.1"

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by reasonerjt** [comment]
@Dannhausen Are you using AWS S3?

**Comment by Dannhausen** [comment]
Yes, but a self hosted privat version.

**Comment by notz** [comment]
Experience the same problem with final 1.14.0 and aws plugin 1.10.0 with OVH S3 storage. Are there any news?

**Comment by oponomarov-tu** [comment]
I confirm the same behaviour on a fresh installation of Velero in a brand new EKS cluster (v1.30).

```text
│     Image:         velero/velero:v1.14.0
│     Image ID:      docker.io/velero/velero@sha256:b871c72cd59908f5ca1ee1690952085b628e010771dba1485f2ed6d8d5e917fe
...
│     Image:          velero/velero-plugin-for-aws:v1.10.0
│     Image ID:       docker.io/velero/velero-plugin-for-aws@sha256:80f84902d9f644aecce043908867a6629cc89b7ddb8c6633ad0e8bbe2c364e7e
```

**Comment by Dannhausen** [comment]
Updated to 1.14.1 with helm chart version 7.1.5, still have this issue

**Comment by rvandernoort** [comment]
I also face these errors on GCP with velero 1.14.1 plugin 1.10.1 chart 7.2.1:
```
time="2024-09-20T07:27:30Z" level=error msg="error encountered while scanning stdout" backup-storage-location=velero/default cmd=/plugins/velero-plugin-for-gcp controller=backup-storage-location error="read |0: file already closed" logSource="pkg/plugin/clientmgmt/process/logrus_adapter.go:90"
```

**Comment by gnomefin** [comment]
what is the stable version matrix for GCP plugin and velero? I followed the guidance from gcp plugin docs, 1.14.x and 1.10.x seems doesn't work.

**Comment by rvandernoort** [comment]
> what is the stable version matrix for GCP plugin and velero? I followed the guidance from gcp plugin docs, 1.14.x and 1.10.x seems doesn't work.

velero 1.14.1 and gcp-plugin 1.10.1 are working fine for me with the exception of the occasional error message as indicated above, but the backups are being made at the right moments.

**Comment by vadlungu** [comment]
We are experience the same issue:

|Chart version: |`8.1.0`|
| ------------- | ----- |
| Velero version    | 1.15.0 | 
| AWS plugin version | v1.11.0 |


<details>
<summary>
velero plugin for aws
</summary>

```sh
 k get pods -o json | jq '.items[].spec.initContainers[].image'
"velero/velero-plugin-for-aws:v1.11.0"
```

</details>



<details>
<summary>
velero server version
</summary>

```sh
k exec -it velero-6c8c466c59-4xg4l -- /velero version
Defaulted container "velero" out of: velero, velero-plugin-for-aws (init)
Client:
        Version: v1.15.0
        Git commit: 1d4f1475975b5107ec35f4d19ff17f7d1fcb3edf-dirty
Server:
        Version: v1.15.0
```

</details>



<details>
<summary>
velero helm chart version
</summary>

```sh
helm list
NAME    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
velero  velero          3               2024-12-16 18:35:35.619604 +0200 EET    deployed        velero-8.1.0    1.15.0
```
</details>


<details>
<summary>
velero error
</summary>

```sh
time="2024-12-17T08:55:43Z" level=error msg="error encountered while scanning stdout" backup-storage-location=velero/default cmd=/plugins/velero-plugin-for-aws controller=backup-storage-location error="read |0: file already closed" logSource="pkg/plugin/clientmgmt/process/logrus_adapter.go:90"
```
</details>

**Comment by gnomefin** [comment]
> > what is the stable version matrix for GCP plugin and velero? I followed the guidance from gcp plugin docs, 1.14.x and 1.10.x seems doesn't work.
> 
> velero 1.14.1 and gcp-plugin 1.10.1 are working fine for me with the exception of the occasional error message as indicated above, but the backups are being made at the right moments.

yeah it does work in GKE, all backup processes were perfect, but the error exists. Could we ignore this error as I update the cluster version over time.

**Comment by e-tip** [comment]
I'm facing this issue too on a EKS cluster, version 1.30, velero version 1.15.2 and aws plugin version 1.11.1. The strange part is that i cannot access logs for a run backup 
```
➜  ~ velero backup logs full-backup-new
➜  ~ 
```

**Comment by natdol** [comment]
Same errors for brand new installation:
EKS 1.30.8
Velero 1.15.2
AWS Plugin : 1.11.1
Helm Chart: 8.4.0

**Comment by x0ptr** [comment]
same errors:
velero client v1.16.1, server v1.16.1
aws plugin v1.12.1

**Comment by prologic** [comment]
Who's working on a fix for this?

**Comment by Hackmeat** [comment]
Same error 
Velero 1.16.2 
Running on AKS with velero-plugin-for-microsoft-azure:v1.12.2 (No AWS Plugin running)

**Comment by jpdias** [comment]
Hello,

Facing the same error after updating from 1.14 to 1.15 on AWS S3 too.

**Comment by g0k8s** [comment]
Hello,

facing the same issue using aws s3 storage: 
- velero helm chart v11.2
- kuvernetes version v1.33.5

**Comment by pisto** [comment]
same spurious (?) error, velero 1.16 and gcp plugin 1.12.1,

**Comment by wagnst** [comment]
same, velero helm chart `12.0.1` and plugin `velero-plugin-for-aws:v1.14.1` using a ceph-based s3 backend

**Comment by andrisrovw** [comment]
We're facing the same error using S3 storage with

- velero version v1.17.0
- kubernetes version v1.34.9
- helm chart version 11.1.1
